### PR TITLE
Pages new clone

### PR DESCRIPTION
**Pull request by erijo**
### What does this PR do?

Updates the documentation to match the changes in #289. Plus an extra fix to document #287.

### What issues does this PR fix or reference?

* #289 
* #287 

### Previous Behavior

Documents the old behavior.

### New Behavior

Documents the new behavior :)

### Have [tests][1] been written for this change?

No

### Have these commits been [signed with GnuPG][2]?

Yes

---

Please review [yadm's Contributing Guide][3] for best practices.

[1]: https://github.com/TheLocehiliosan/yadm/blob/master/.github/CONTRIBUTING.md#test-conventions
[2]: https://help.github.com/en/articles/signing-commits
[3]: https://github.com/TheLocehiliosan/yadm/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: _docs/030_getting_started.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ Clone your existing repo using yadm.
...
     yadm status
 
-The `clone` command will attempt to `merge` your existing repository, but if it
-fails, it will `stash` any conflicting data. See
-[this question](faq#i-just-cloned-my-repository-and-conflicting-data-was-overwritten-why)
-in the FAQ if you need help.
+The `clone` command will attempt to check out all files that exists in the
+repository. If a file already exists locally and have content that differs from
+the one in the repository, the local file will be left unmodified and you'll
+have to review and resolv the difference.
```
Comment (rasa) [inline _docs/030_getting_started.md:28]:
```suggestion
The `clone` command will attempt to check out all files that exist in the
repository. If a file already exists locally and has content that differs from
the one in the repository, the local file will be left unmodified and you'll
have to review and resolve the differences.
```

File: _docs/110_faq.md [hunk 2]
```diff
@@ -8,17 +8,6 @@ permalink: /docs/faq
 
 ## Usage
 
-### I just cloned my repository and conflicting data was overwritten. Why?
```
Comment (rasa) [inline _docs/110_faq.md:11]: Why remove this section? It is still relevant.

File: _docs/110_faq.md [hunk 3]
```diff
@@ -45,15 +34,6 @@ yadm gitconfig "user.name"  "Alternate Name"
 Note: Configuring these settings directly in the yadm repo will require you
 to configure it each time you clone the repo.
 
-### Why was I prompted to login twice while cloning my repo?
```
Comment (rasa) [inline _docs/110_faq.md:48]: Why remove this section? It is still relevant.

**Review by rasa — CHANGES_REQUESTED** [review]

File: _docs/110_faq.md (hunk 2 above)
Comment (TheLocehiliosan) [reply _docs/110_faq.md:11]: The new version of clone does not stash differences anymore. It merely leaves the files as modified.

File: _docs/110_faq.md (hunk 3 above)
Comment (TheLocehiliosan) [reply _docs/110_faq.md:48]: The new version of clone doesn't need to inspect the default branch from the remote anymore. So the possible double login experience is gone.

File: _docs/030_getting_started.md (hunk 1 above)
Comment (erijo) [reply _docs/030_getting_started.md]: Thanks, fixed now!

**Pull request merged**